libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -308,6 +314,19 @@ export const appStrings = {
     </UiString>
   ),
 
+  instructionsBmdBallotNavigationMarkScanPatDevice: () => (
+    <UiString uiStringKey="instructionsBmdBallotNavigationMarkScanPatDevice">
+      When voting with the text-to-speech audio, use your personal assistive
+      device to navigate your ballot. To navigate through contest choices, use
+      the move input. To select or unselect a contest choice as your vote, use
+      the select input. To navigate through the contests, use the move input to
+      navigate to the controls labelled "next" and "back" and use the select
+      input to navigate to a new contest. To get started, use the move input to
```

Comment:
Are "to navigate through the contest" and "to navigate to a new contest" redundant? Or maybe subtly different because the latter informs the voter that using the "next" and "back" buttons are the specific actions to take in order to perform the former?

Reply:
Also, I can't recall if focus will jump to "Next" when a voter is using both PAT input and speech-to-text. Do you think it's worth mentioning here if so?

Reply:
Yeah, good point WRT the autofocus behaviour - will try to write something up that doesn't sound too confusing. Will also see if I can get reword to get rid of the repeated "to navigate"